src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -6323,6 +6323,86 @@ def test_aks_update_with_defender(self, resource_group, resource_group_location)
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @live_only()
```

Comment:
May I ask why do you need to mark these new tests as `@live_only()`?

Reply:
its failing with the following error:

 "Content": "|Type|Test Case|Error Message|Line|\n|---|---|---|---|\n|Failed|test_aks_create_with_azuremonitormetrics|E   azure.cli.testsdk.exceptions.CliExecutionError: The CLI throws exception CLIError during execution and fails the command.<br><br>During handling of the above exception, another exception occurred:<br>E   knack.util.CLIError: Please run 'az login' to setup account.|azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:6325|\n|Failed|test_aks_update_with_azuremonitormetrics|E   azure.cli.testsdk.exceptions.CliExecutionError: The CLI throws exception CLIError during execution and fails the command.<br><br>During handling of the above exception, another exception occurred:<br>E   knack.util.CLIError: Please run 'az login' to setup account.|azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:6361|\n"


I'm going to be adding the livetest only again as it looks like it needs to run az login to execute properly.

Reply:
It seems we could remove the `@live_only()` tag. For the error about "run 'az login' to setup", that's caused by you'll need to add recording files for these 2 cases, otherwise, the check-in CI would try to run the case in live mode but fail due to it's not configured to be logged in, so it cannot create azure resources.

As for the recording file, the live test pipeline would upload the recording files to artifact once the cases passed the live test. But case `test_aks_create_with_azuremonitormetrics` always fail, I think it's caused by the PUT request sent by addonput.py/addon_put is still ongoing (doesn't poll until it's done), and you send another delete request (line 6358~6360) resulting in a conflict. Maybe you could use the `az aks wait` command to poll until the previous put request is done as a fix.